### ISSUE TITLE
update url's with category clicks so that specific url can be sharable

### ISSUE THREAD

**Issue by paulchrisluke** [whynotearth/foodouken.com#57]
instead of keeping all clicks on "foodouken.com" allow slugs to be in the url so people can share.

format should probably be

foodouken.com/`tenent`/`category`/`slug`

**Comment by paulchrisluke** [comment]
@mort3za you have experience with this correct?

**Comment by mort3za** [comment]
Yeah, seems easy, just use vue router and create new pages and routes. Is there any problem?

**Comment by paulchrisluke** [comment]
@GerardBalaoro is currently working on this. We need to add /thanks to our success screen as well, for conversion tracking.

**Comment by GerardBalaoro** [comment]
@paulchrisluke I'm not working on it currently. Still waiting for @atharva3010 to update API calls to the new endpoints, I'll gladly take over the categories page when its ready. This is just an implementation of [nested routes](https://router.vuejs.org/guide/essentials/nested-routes.html).

**Comment by paulchrisluke** [comment]
sounds good, I was just cleaning up our project board and saw our discussion in discord. Trying to launch today, happy if you'd be able to do this work once Atharva is finished

**Comment by paulchrisluke** [comment]
closing since this request changed a bunch. Made new issue  https://github.com/whynotearth/foodouken.com/issues/142
